Bio/SeqFeature.py

```diff
@@ -925,6 +925,25 @@ def __radd__(self, other):
             return self._shift(other)
         else:
             return NotImplemented
+    
+    def __sub__(self,other):
+        """Subtracting an integer will give a new FeatureLocation with
```

Comment:
This docstring needs to start with a one line summary ending with a full stop, then a blank link. See PEP257 and the associated checks for this in flake8.